iam-login-service/src/main/java/it/infn/mw/iam/api/client/service/DefaultClientService.java

```diff
@@ -107,6 +107,13 @@ public ClientDetailsEntity updateClient(ClientDetailsEntity client) {
     return clientRepo.save(client);
   }
 
+  @Override
+  @CacheEvict(cacheNames = DefaultScopeMatcherRegistry.SCOPE_CACHE_KEY, key = "{#clientId}")
```

Comment:
I would use the same key as `updateClient()`

Reply:
Done. Using same key as `updateClient()` now. Thanks.

Reply:
We can avoid the cache eviction here because client caching is currently only related to scope matchers.